src/foam/flow/DocumentationFolderDAO.js

```diff
@@ -35,31 +35,18 @@ sink = prepareSink(sink);
 foam.dao.Sink         decorated = decorateSink_(sink, skip, limit, order, predicate);
 foam.dao.Subscription sub       = new foam.dao.Subscription();
 
-java.nio.file.FileSystem fs = java.nio.file.FileSystems.getDefault();
-java.nio.file.DirectoryStream<java.nio.file.Path> contents;
-
-try {
-  contents = java.nio.file.Files.newDirectoryStream(fs.getPath(getDir()), "*.flow");
-} catch ( java.io.IOException e ) {
-  throw new RuntimeException(e);
-}
-
-for ( java.nio.file.Path path : contents ) {
-  if ( sub.getDetached() ) break;
-
+java.nio.file.DirectoryStream<java.nio.file.Path> paths = new foam.nanos.fs.ResourceStorage(getDir()).getDirectoryStream("", "*.flow");
```

Comment:
We could make a composite UnionStorage class which composes two Storage classes together.  Then we can just use an instance of UnionStorage for our Storage class in the DocumentationFolderDAO.

UnionStorage would read from both child storages when enumerating the directory and unify them, and it would only write to one of them.  Letting us read fro mthe JAR and the documentation folder, bu tonly write to the documentation folder.

As it stands this won't work when we don't run from a JAR i believe.